src/views/GraphView/components/GraphComponent/GraphOptionsPanel/GraphOptionsPanel.tsx

```diff
@@ -216,7 +214,8 @@ function GraphOptionsPanel(props) {
                     checked={
                         graphOptions.linksLegend
                         && graphOptions.linksColor
-                        && !linkLegendDisabled}
+                        && !linkLegendDisabled
+}
```

Comment:
same autofix weirdness here

Reply:
fixed in [3ae605b](https://github.com/bcgsc/pori_graphkb_client/pull/51/commits/3ae605b4002a665a6ea1e848c37f3bb098e6fec5)